content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
 kubectl create cm dex --from-file=config.yaml=dex-config.yaml --dry-run -oyaml | kubectl apply -f -
 ```
 
+#### Log in with LDAP / Active Directory
+
+As you saw in the overview, we use [Dex](https://github.com/dexidp/dex) for providing user authentication.
+Dex supports several authentication methods:
+
+* Static Users (what we used so far)
+* LDAP / Active Directory
+* External IdP (eg Google, LinkedIn, Github, ...)
+
+This section focuses on setting up Dex to authenticate with an existing LDAP database.
+
+1. **[OPTIONAL]** If you don't have an LDAP database, you can set up one following these instructions:
```

Comment:
Change the square brackets to round brackets (parentheses), as square brackets usually indicate some coding convention or other. Also, it's not necessary to use caps as well as bold text - it may seem over-emphatic to the reader:

```
***(Optional)***
```